### DISCUSSION TITLE
Re-position Control Center to Better Align with Taskbar at Top

### DISCUSSION THREAD

**Discussion by PandaBoyo** [valinet/ExplorerPatcher#1135] [Ideas]
Would it be possible to change the position of the windows 11 control center so that it appears directly under the taskbar when using top-of-screen alignment, like how the windows 11 menu appears?

Current Position:
![Control Center](https://user-images.githubusercontent.com/121266577/226196758-c9fba52f-5d15-4f68-8a29-d01a7834152b.png)

Crude Photoshop of New Position:
![Control Center (new)](https://user-images.githubusercontent.com/121266577/226197077-f2cfa042-e707-47c4-a88e-cc8b380cad23.png)

**Comment by valinet** [comment]
Probably, but tbh, after all the glitches and issues with positioning Start correctly, I'd rather Microsoft do it. It's just another avenue for this software failing in the future...
